public_html/templates/index.html

```diff
@@ -7,116 +7,136 @@
 			<span>
 				<a class="reviewer-link" href="{{ row.reviewed_by_url }}" target="_blank">{{ row.status_user }}</a>
 			</span><br>
-			<span class="reviewer-timestamp">{{ row.review_timestamp }}</span>
+			<time datetime="{{ row.review_timestamp }}" class="reviewer-timestamp">{{ row.review_timestamp }}</time>
```

Comment:
Using `<time>` when something is strictly displaying a time; the `datetime` attribute tells the browser this is a structured datetime string, so it can be used or transformed; e.g. browser extensions that convert times, adding something to your calendar (which obviously they won't do here)